service/history/transferQueueProcessor_test.go

```diff
@@ -205,9 +209,14 @@ workerPump:
 		select {
 		case task := <-tasksCh:
 			if task.TaskType == persistence.TransferTaskTypeDecisionTask {
-				s.mockMatching.On("AddDecisionTask", mock.Anything, createAddRequestFromTask(task, 0)).Once().Return(nil)
+				s.mockMatching.On("AddDecisionTask", mock.Anything, createAddRequestFromTask(task, wtimeout)).Once().Return(nil)
 				if task.ScheduleID == firstEventID+1 {
-					s.mockVisibilityMgr.On("RecordWorkflowExecutionStarted", mock.Anything).Once().Return(nil)
+					s.mockVisibilityMgr.On("RecordWorkflowExecutionStarted", mock.MatchedBy(
+						func(request *persistence.RecordWorkflowExecutionStartedRequest) bool {
+							fmt.Println("Test TTL: ", wtimeout)
```

Comment:
maybe use s.logger.Info instead

Reply:
Haha, I had this for local testing and forgot to remove. Removing.